src/remove-search-json.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const fs = require('fs');
```

Comment:
In some places you're using `fs-extra`, but in others, not...?

Reply:
Not a huge deal, just curious.

Reply:
I'm only using it where required. I only really needed it in that one file. Bobby convinced me to use it for the BOM handling.

Reply:
I would suggest consistently using `fs-extra`.